tests/conftest.py

```diff
@@ -15,14 +57,12 @@ def collect(self):
         test_dir = os.path.dirname(str(self.fspath))
         for platform, archs in spec["Output Files"].items():
             for arch, filename in archs.items():
-                filepath = os.path.join(test_dir, filename)
-                if os.path.exists(filepath):
-                    yield FLOSSTest(self, platform, arch, filename, spec)
-
-
-def extract_strings(sample_path):
-    # TODO: implement me
-    return []
+                # TODO only test Windows PE files for now
+                _, fileext = os.path.splitext(filename)
+                if fileext == ".exe":
```

Comment:
also, i think these are items that should be marked `xfail` see #76, rather than ignored. this indicates we'd like to support the functionality, but that it not expected to work yet.